.github/workflows/setup_pwa_manual.yml

```diff
@@ -0,0 +1,137 @@
+name: SalesforceCommerceCloud/pwa-kit/e2e
+on:
+  workflow_dispatch:
+    inputs:
+      use_extensibility:
+        type: boolean
+        description: Use Extensibility?
+        default: true
+      project_name:
+        type: string
+        description: Project Name
+        default: "scaffold-pwa"
+      instance_url:
+        type: string
+        description: Instance Url
+        default: "https://zzrf-002.dx.commercecloud.salesforce.com"
+      org_id:
+        type: string
+        description: Org Id
+        default: "f_ecom_zzrf_002"
+      short_code:
+        type: string
+        description: Short Code
+        default: "kv7kzm78"
+      client_id:
+        type: string
+        description: Client Id
+      site_id:
+        type: string
+        description: Site Id
+        default: "RefArch"
+      mrt_target_env:
+        type: string
+        description: MRT Targent Environment
+      is_private_client:
+        type: boolean
+        description: Is Private Client?
+        default: true
+      setup_hybrid:
+        type: boolean
+        description: Setup Phased Headless rollout?
+        default: false
+
+jobs:
+  setup-pwa-kit:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set environment variable
+        run: |
+          set -e
+          echo "PROJECT_DIR=my-retail-react-app" >> $GITHUB_ENV 
+          url="${{ github.event.inputs.instance_url }}"
+          hostname=$(echo "$url" | awk -F '://' '{split($2, parts, "/"); print parts[1]}')
+          echo "HOSTNAME=$hostname" >> $GITHUB_ENV
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18
+          cache: 'npm'
+
+      - name: Setup PWA dependencies
+        run: |-
+          set -e
+          sudo apt-get update -yq && sudo apt-get install --no-install-recommends python2 python3-pip time -yq
+          sudo pip install -U pip setuptools
+          sudo pip install awscli==1.18.85 datadog==0.40.1
+          node ./scripts/gtime.js monorepo_install npm ci
+          npm ci
+        shell: bash
```

Comment:
There is an existing action called `setup_ubuntu` that seems to share some of the similar dependencies. Should we be reusing that or refactoring that action so these steps can be used independently?

Reply:
the action `setup_ubuntu` installs other packages like snyk, lighthouse and is also reporting bundle size which is not required for this workflow